index.html

```diff
@@ -166,16 +166,17 @@ <h2 class="section-title">Contact me</h2>
             If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
             I’d love to help with it!
           </p>
-          <form action="https://formspree.io/f/mayzvopn" method="POST">
+          <form action="https://formspree.io/f/mayzvopn" method="POST" id="contact-me-form">
             <div class="form-element-container">
               <input type="text" class="form-element" name="name" placeholder="Write your name here" maxlength="30" title="You can only add maximum: 30 characters" required>
             </div>
             <div class="form-element-container">
-              <input type="email" class="form-element" name="email" placeholder="Write your Email here" title="example@email.com" pattern="^[^@\s]+@[^@\s]+[.a-z]+$" required>
+              <input type="email" class="form-element" id="user-email" name="email" placeholder="Write your Email here" title="example@email.com" pattern="^[^@\s]+@[^@\s]+[.a-z]+$" required>
             </div>
```

Comment:
Carmen you have done a great job in this but I have a small suggestion for you, next time try to isolate the javascript patterns from the input email to the js file you have, so that keeps your code more maintainable and more organized